src/components/ui/UserPicture.vue

```diff
@@ -80,13 +80,12 @@ export default {
       }
     },
     buildSrcset() {
+      if (!this.slot) return '';
       return [40, 100, 264]
-        .map(
-          (s) =>
-            `${this.avatar.picture}?size=${s} ${(s / this.slot).toPrecision(
-              2,
-            )}x`,
-        )
+        .map((s) => {
+          const res = (s / this.slot).toPrecision(2);
```

Comment:
So, this will still return "NaN" if ```this.slot``` is invalid. What should we do when we get NaN?

Reply:
Actually looking closer at the code, I don't think we need to check to see if this.slot is 0/invalid/etc. since it's not a prop and we set it in the `updateSize` method, so it will only ever be 40 or 100 or 264.

Reply:
I'd simplify this overall.
Just have something like:
```js
const SRC_SETS = {
  "40": [[40, "1x"], [100, "2x"]],
  "100": [[100, "1x"], [268, "2x"]],
  "264": [[264, "1x"], [528, "2x"]],
}
```
And don't do the math. +1 on using constants for the sizes.